web/satellite/src/components/project/UsageReport.vue

```diff
@@ -107,12 +109,20 @@
             return this.$store.state.usageModule.projectUsage.objectCount.toPrecision(5);
         }
 
-        public mounted(): void {
-            this.$store.dispatch(PROJECT_USAGE_ACTIONS.FETCH_CURRENT_ROLLUP);
+        public async mounted(): Promise<void> {
+            try {
```

Comment:
Are you sure that we can make ``mounted`` async and make it await API execution?

Reply:
Previous implementation was not awaited and simply refresh store and ui values after API response

Reply:
Yes, this was intentional. It does the same thing but fit linter demands